backend/dal.keycloak/Partials/PimsKeycloakUserService.cs

```diff
@@ -129,20 +131,32 @@ public async Task<Entity.User> UpdateUserAsync(Entity.User user)
             // Update PIMS
             _mapper.Map(user, euser);
 
-            // Update Roles.
-            addRoles.ForEach(async r =>
+            // Remove all keycloak groups from user.  // TODO: Only add/remove the ones that should be removed.
```

Comment:
If I had more time I would only remove the ones that need to be removed, but for simplicity I'm removing them all before adding back the relevant ones.